### PR TITLE
Grunt

### PR DESCRIPTION
**Pull request by jacetchan**
If merged, this pull request will:
- [Serve all JS bundles and CSS files from S3 and create a build process.](https://trello.com/c/8kHxnOx4)
- [Update README.md with step-by-step instructions](https://trello.com/c/WHvrTqZP)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +28,13 @@
   "homepage": "https://github.com/project-nomad/proxy-jacetchan#readme",
   "dependencies": {
     "express": "^4.16.3",
+    "grunt": "^1.0.3",
```
Comment (nataliestam) [inline package.json]: should grunt be a devDependency?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+
+## Instructions
+1. Create a folder on a local directory called `project-nomad`.
+2. Navigate into `project-nomad`. For each of the following modules: `booking-module`, `listing-description-module`, `reviews-module`, `image-carousel-module`, clone their respective repos.
+3. For each of the modules listed in Step 2, follow the instructions listed in their respective READMEs in order to seed their database. Finish this step before moving on.
+4. Navigate into `project-nomad` and clone this repo: `git clone https://github.com/project-nomad/proxy-jacetchan.git`.
+5. Navigate into `proxy-jacetchan` and run `npm install`.
+6. Run `npm run react-dev-all`. This will run the `babel` with the `--watch` for each microservice.
+7. Create a new terminal instance in order to proceed with the next steps.
+8. Run `grunt concat`. This will create _one_ `bundle.js` sourced from each module's bundle file.
```
Comment (nataliestam) [inline README.md]: could all 3 grunt commands combined into one command?

File: package.json (hunk 1 above)
Comment (jacetchan) [reply package.json]: Just made a [ticket](https://trello.com/c/Wkhtwi0m) for it! 👍

File: README.md (hunk 2 above)
Comment (jacetchan) [reply README.md]: Yes, definitely! I've gone ahead and [made a ticket](https://trello.com/c/sVk8nVm5) to address it! :octocat:

**Comment by jacetchan** [comment]
Thanks! I'll make those changes with the next iteration! :)

**Pull request merged**